parser/src/test/java/com/microsoft/gctoolkit/parser/unittests/UnifiedGenerationalParserTest.java

```diff
@@ -65,4 +65,32 @@ public void testCMSLogsMissingTimeStamps() {
             //  0,   1,    2,     3,     4,     5,     6,    7,    8,     9,      10,     11,      12, 13, 14, 15, 16, 17
             {   0,   0, 6167,     0,     0,     0,     0,    0,    0,     0,       0,      2,       2,  2,  2,  2,  2,  2},
     };
+
+    //Serial
+    @Test
+    public void testSerialLogs() {
+        int i = 0;
+        for (String name : serial) {
+            try {
+                Path path = new TestLogFile("serial/" + name).getFile().toPath();
+                TestResults testResults = testGenerationalSingleLogFile(path);
+                analyzeResults(name, testResults, serialNumberOfDifferentCollectors[i], serialCounts[i++]);
+            } catch (IOException ioe) {
+                fail(ioe.getMessage());
+            }
+        }
+    }
+
+    private static final String[] serial = {
+            "factorization-serialgc-tip.log"
+    };
+
+    private static final int[] serialNumberOfDifferentCollectors = {
+            2,
+    };
+
+    private static final int[][] serialCounts = {
+            //  0,   1,    2,     3,     4,     5,     6,    7,    8,     9,      10,     11,      12, 13, 14, 15, 16, 17
+            {   0,  13,    0,     0,     0,     0,     0,    0,    0,     0,       4,      0,       0,  0,  0,  0,  0,  0},
```

Comment:
A new developer coming into this will struggle, can we add a comment stating what this array effectively tests?